hardware_interface/test/fake_components/test_generic_system.cpp

```diff
@@ -244,16 +244,18 @@ class TestGenericSystem : public ::testing::Test
     <joint name="joint1">
       <command_interface name="position"/>
       <command_interface name="velocity"/>
-      <state_interface name="position"/>
+      <state_interface name="position">
+        <param name="initial_value">3.45</param>
```

Comment:
It feels like I'm only seeing half of the changes to the tests. The change in format is great but there should be now an assertion that the initial_value fields actually take effect when using a fake system, right?

Reply:
This feature is already added to the parser in #581

Reply:
I didn't mean the parser. Bluntly put: I'd like to see a test or two.

Reply:
But they are here. The tests are the same, but with changed URDF. They are working as before, which means we are all good. If you remove here changed lines, the tests will fail be because they expect initial positions.

Reply:
You are right, thanks for the explanation!